hydra-access-controls/lib/active_fedora/accessible_by.rb

```diff
@@ -11,7 +11,7 @@ def accessible_by(ability, action = :index)
       when :update, :edit, :create, :new, :destroy then [:edit]
     end
 
-    filters = gated_discovery_filters(permission_types, ability).join(" OR ")
+    filters = gated_discovery_filters(permission_types, ability).reject(&:blank?).join(" OR ")
```

Comment:
Note: i thought this would be required.  But if you look at CI, it was not required.